IdentityCore/src/util/MSIDKeyOperationUtil.h

```diff
@@ -0,0 +1,53 @@
+//
+//  MSIDKeyOperationUtil.h
+//  IdentityCore
+//
+//  Created by Ameya Patil on 9/24/21.
+//  Copyright © 2021 Microsoft. All rights reserved.
+//
+
+#import "MSIDJwtAlgorithm.h"
+NS_ASSUME_NONNULL_BEGIN
+@interface MSIDKeyOperationUtil : NSObject
+
+- (instancetype)init NS_UNAVAILABLE;
+
++ (instancetype)new NS_UNAVAILABLE;
+
++ (instancetype)sharedInstance;
+
+/// Returns if supplied key is from secure enclave
+/// @param key key to check
+- (BOOL)isKeyFromSecureEnclave:(_Nonnull SecKeyRef)key;
```

Comment:
nit - since you've added NS_ASSUME_NONNULL_BEGIN prefix above, you don't need _Nonnull here

Reply:
Had not put it before but put it in later.